perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -134,6 +144,28 @@ public static boolean authorized(PerunSession sess, String policyDefinition, Lis
 		return resolveAuthorization(sess, policyRoles, mapOfBeans);
 	}
 
+	public static boolean authorizedToManageRole(PerunSession sess, PerunBean object, String roleName) throws RoleManagementRulesNotExistsException {
```

Comment:
Static method should always have javadoc.

Reply:
Added